### ISSUE TITLE
[ACCESSIBLITY][Screen Reader - Overview]:In Browse mode, NVDA focus goes out of Graphic Slider instead of moving to next section of Slider.

### ISSUE THREAD

**Issue by Shubham786786** [highcharts/highcharts#15440]
**User Experience:**
Screen Reader user will be impacted here because they are not able to navigate to different section of Slider graph hence find difficulty in grab other slider section information.

**Test Environment:**
OS: Windows 10 Version Dev (OS Build 21327.1010)
Browser: Microsoft Edge Dev Build 91.0.838.3
URL: [https://www.highcharts.com/demo/bar-stacked](url)

#### Expected behaviour
In Browse mode, NVDA focus should not goes to outside control when user navigate using 'Down' arrow key, NVDA focus should goes to next section of Slider i.e From Green to Black to Blue section of slider.

#### Actual behaviour
In Browse mode, NVDA focus goes out of Graphic Slider instead of moving to next section of Slider, Reading order doesn't match visual look.

#### Live demo with steps to reproduce
1. Open above URL on Anaheim Dev Browser.
2. Open NVDA and Turn NVDA browse mode using 'CapsLock + Space' key.
3. Navigate to ‘Graph’ Slider using 'down' arrow key.
5. Observe the issue.

WCAG Guidelines:
[https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence](url)

**Attachments**
[Attachment.zip](https://github.com/highcharts/highcharts/files/6262194/Attachment.zip)

**Comment by oysteinmoseng** [comment]
Hi @Shubham786786, this is a limitation of the SVG layout of using multiple series in Highcharts. Highcharts will always contain data points in the same series in the same SVG group, which means you will navigate each data point in the series sequentially with a screen reader.

We are looking at implementing custom navigation that allows for navigating multiple dimensions with screen readers, but for now we are mostly limited to single dimension navigation.

If you prefer navigating all the points in one stack first, and then moving on to the next stack, the best workaround is to keep all of the points of the stack in the same series. This is not very convenient, but does get the behavior that you are looking for.

Here is an example of doing this: https://jsfiddle.net/qzk03nsd/ 

Hope this is of help as a workaround for now.
